crates/ibc/src/applications/transfer/relay/on_recv_packet.rs

```diff
@@ -7,7 +7,7 @@ use crate::core::ics04_channel::handler::ModuleExtras;
 use crate::core::ics04_channel::packet::Packet;
 use crate::prelude::*;
 
-pub fn process_recv_packet_execute<Ctx: TokenTransferExecutionContext>(
+pub fn process_recv_packet<Ctx: TokenTransferExecutionContext>(
```

Comment:
now that we'll have a `process_recv_packet_validate`, this should be `process_recv_packet_execute`